dev/com.ibm.ws.security.spnego/resources/com/ibm/ws/security/spnego/internal/resources/SpnegoMessages.nlsprops

```diff
@@ -113,3 +113,8 @@ SPNEGO_CAN_NOT_GET_DELEGATE_SERVICE_SPN.useraction=Verify the service principal
 SPNEGO_DELEGATE_SPN_LOGIN_TO_KDC_FAILURE=CWWKS4322E: Delegate SPN {0} can not authenticate to the Key Distribution Center (KDC) using the Kerberos configuration file {1} and keytab file {2}. Login exception was received: {3}
 SPNEGO_DELEGATE_SPN_LOGIN_TO_KDC_FAILURE.explanation=Can not authenticate to the Key Distribution Center (KDC).
 SPNEGO_DELEGATE_SPN_LOGIN_TO_KDC_FAILURE.useraction=Verify the service principal name format, and make sure it exists on the Windows domain and that the Kerberos keytab file contains the service principal name.
+
+SPNEGO_AUTHENTICATION_ERROR_PAGE=CWWKS4323E: <html><head><title>SPNEGO authentication failed. Contact your system administrator to resolve the problem.</title></head> \
+<body>The client sent the SPNEGO token. The server cannot validate the client's SPNEGO token.</body></html>.
+SPNEGO_AUTHENTICATION_ERROR_PAGE.explanation=SPNEGO authentication failed. Check the server message.
```

Comment:
What is "the server message" in this context? Can we offer any common reasons that SPNEGO auth fails?

Reply:
There is no common reasons for the server message.

Reply:
Is the "server message" in the logs? The initial message says to contact system admin, the explanation says to check the server message- will the user know what message this refers to?

Reply:
The server message in the logs - yes.
In these problems, there is nothing an end user can do to fix the problem. They have to call the Liberty system administrator. The administrator need to look in the server message

Reply:
Ok- is it possible or helpful to say more specifically where the admin should look for a message- eg system out, trace log,  or more generally- server logs. Or will it be different depending on log configuration? 

If "server message" is an expected term in this context that's ok. But the [tWAS SPNEGO troubleshooting topic](https://www.ibm.com/support/knowledgecenter/SSEQTP_8.5.5/com.ibm.websphere.base.doc/ae/rsec_SPNEGO_troubles.html) doesn't use this term, instead referring to server logs, trace log etc

Reply:
I see we these in Liberty - If you search for "server message", "server message.log"  "server logs"... I think you want us to us the server logs?

Reply:
I think "server logs" would include message, trace, etc if there's not a more specific log to call out. So

`SPNEGO authentication failed. For more information, check the server log messages.`

Reply:
Done